### ISSUE TITLE
test_ewsdatetime.EWSDateTimeTest fails with `Items in the first set but not the second`

### ISSUE THREAD

**Issue by hashworks** [ecederstrand/exchangelib#1116]
**Describe the bug**
When building the AUR package [python-exchangelib](https://aur.archlinux.org/packages/python-exchangelib) with an updated [PKGBUILD](https://fb.hash.works/sOrV34aq/) the test `tests.test_ewsdatetime.EWSDateTimeTest` fails with `Items in the first set but not the second:
'Europe/Kyiv'`.

**To Reproduce**
Best case, just run `python setup.py test`. But I suspect the error comes from newer dependency versions in Arch Linux.

In that case, on Arch Linux:
```
git clone https://aur.archlinux.org/python-exchangelib.git
cd python-exchangelib
wget -O PKGBUILD https://fb.hash.works/sOrV34aq
extra-x86_64-build
```

**Expected behavior**
The tests are green, even with the latest dep versions.

**Log output**
```
======================================================================
FAIL: test_ewstimezone (tests.test_ewsdatetime.EWSDateTimeTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/python-exchangelib/src/exchangelib-4.7.6/tests/test_ewsdatetime.py", line 68, in test_ewstimezone
    self.assertEqual(set(sanitized) - set(EWSTimeZone.IANA_TO_MS_MAP), set())
AssertionError: Items in the first set but not the second:
'Europe/Kyiv'

----------------------------------------------------------------------
Ran 60 tests in 0.153s

FAILED (failures=1, skipped=24)
```

**Additional context**
* python-3.10.7-1
* python-cached-property-1.5.2-5
* python-cryptography-38.0.1-1
* python-defusedxml-0.7.1-4
* python-dnspython-1:2.2.1-1
* python-isodate-0.6.1-1
* python-lxml-4.9.1-1
* python-requests-kerberos-0.14.0-2
* python-requests-ntlm-1.1.0-8
* python-requests-oauthlib-1.3.1-1
* python-pytz-2022.1-1
* python-future-0.18.2-7
* python-tzlocal-1:2.1-2
* python-setuptools-1:62.3.4-1
* python-pip-22.2.2-2
* python-requests-mock-1.10.0-1
* python-yaml-6.0-1
* python-dateutil-2.8.2-4
* python-psutil-5.9.1-1

**Comment by thrasibule** [comment]
fixed here: d5d386f54adec8ab02f871332b89e1176c214ba2

**Comment by ecederstrand** [comment]
FYI, I've moved these inherently flaky tests into the integration test suite which is not run by Linux package repos. Hopefully that will fix this situation from reappearing.
